sdks/python/apache_beam/runners/portability/flink_runner.py

```diff
@@ -29,7 +32,12 @@
 
 class FlinkRunner(portable_runner.PortableRunner):
   def default_job_server(self, options):
-    return job_server.StopOnExitJobServer(FlinkJarJobServer(options))
+    flink_master_url = options.view_as(FlinkRunnerOptions).flink_master_url
```

Comment:
This should be `flink_master` to be compliant with https://github.com/apache/beam/blob/14b5ef8b3e63d77f5f34730ce10b9ed581380ffa/runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java#L66

Reply:
Sorry, I merged before refreshing and seeing your comments. https://github.com/apache/beam/pull/9803